components/suggest/src/schema.rs

```diff
@@ -36,7 +36,7 @@ CREATE TABLE keywords(
 -- The FK to suggestion_id makes it so full keywords get deleted when the parent suggestion is deleted.
```

Comment:
Nit: Would you mind dropping this comment, since we aren't using `ON DELETE CASCADE` anymore?